### PR TITLE
PERF-3576 Add a script to analyze local CedarMetrics output

### PR DESCRIPTION
**Pull request by cswanson310**

### CODE REVIEW

File: src/workloads/contrib/analysis/test_result_summary.py [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+        },
+        'second metric example': {...},
+        ...
+    }
+    """
+    results = {}
+    for (metric_name, diffed_readings) in metrics_of_interest.items():
+        if diffed_readings == []:
+            print(
+                "No measurements to summarize for %s metric for %s. Skipping" % (metric_name, actor_name))
```
Comment (ccstolley) [inline src/workloads/contrib/analysis/test_result_summary.py:119]: `actor_name` not in scope here

**Review by ccstolley — APPROVED** [review]
I left 1 comment and I think you can drop `requirements.txt` entirely. This is really useful, thanks!

File: src/workloads/contrib/analysis/test_result_summary.py (hunk 1 above)
Comment (cswanson310) [reply src/workloads/contrib/analysis/test_result_summary.py:119]: Good catch! Fixed.

**Comment by cswanson310** [comment]
evergreen merge

**Pull request merged**